src/pages/index.js

```diff
@@ -5,6 +5,8 @@ import Card from "../components/Card.js";
 import "./index.css";
 import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
+
 const config = {
```

Comment:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.